editor/store/selectors.js

```diff
@@ -300,6 +300,40 @@ export function isEditedPostEmpty( state ) {
 	);
 }
 
+/**
+ * Returns true if the post can be autosaved, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
```

Comment:
JSDoc standards treat `@param` and `@return` as separate groupings with newline between, no alignment necessary between groups.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

Reply:
right, correcting